src/Common/tests/CoreFx.Private.TestUtilities/System/AssertExtensions.cs

```diff
@@ -21,6 +21,12 @@ public static void Throws<T>(Action action, string expectedMessage)
             Assert.Equal(expectedMessage, Assert.Throws<T>(action).Message);
         }
 
+        public static void ThrowsContains<T>(Action action, string expectedMessageContent)
+            where T : Exception
+        {
+            Assert.Contains(expectedMessageContent, Assert.Throws<T>(action).Message);
```

Comment:
I've generally been guarding any exception message tests with the culture being en-US (though many of mine come from Win32 FormatMessage).  Should this have a culture guard in it, or indicate that it's only applicable to all-culture message testing? (e.g. the bad parameter value is in the message)

Reply:
Wouldn't the parameter value be found within the message in every culture?